tests/tt_metal/tt_metal/unit_tests/multichip/basic_eth_kernels.cpp

```diff
@@ -186,6 +190,7 @@ bool writer_kernel_no_receive(
 }
 
 TEST_F(N300DeviceFixture, EthKernelsNocReadNoSend) {
+    using namespace CMAKE_UNIQUE_NAMESPACE;
```

Comment:
Yes. Let's say we have two files which are being merged in unity build, and both of them define the same function within CMAKE_UNIQUE_NAMESPACE. If both of them do using namespace at the file level, it would result in both functions being visible at the same time and the call would be ambiguous.